src/node_mgmt/commands.py

```diff
@@ -211,6 +212,61 @@ def readall(s):
     return result
 
 
+def holykell_sensor_address(node):
+    from time import sleep
+    import minimalmodbus
```

Comment:
Yeah I think it's probably better to move this to the top of the file (and also for the other cases where we have imports inside the function).